src/main/java/io/jenkins/plugins/aws/global_configuration/CredentialsAwsGlobalConfiguration.java

```diff
@@ -248,8 +242,8 @@ public static CredentialsAwsGlobalConfiguration get() {
     public ListBoxModel doFillRegionItems() {
         ListBoxModel regions = new ListBoxModel();
         regions.add("Auto", "");
-        for (Regions s : Regions.values()) {
-            regions.add(s.getDescription(), s.getName());
+        for (Region s : Region.regions()) {
+            regions.add(s.id(), s.id());
```

Comment:
Technically a change in behavior, as observed in the UI which now lacks region descriptions. Perhaps we could get these from the EC2 `DescribeRegions` API, although that requires read access. Or maybe this does not matter and we could dispense with displaying the region descriptions in the UI?